ci/run_verilator_pytest.sh

```diff
@@ -14,16 +14,19 @@ SW_BUILD_PATH="${SW_BUILD_PATH:-$SW_BUILD_DEFAULT}"
 
 BOOT_ROM_TARGET="boot_rom/boot_rom.vmem"
 
-TEST_TARGETS=("tests/flash_ctrl/flash_test.vmem"
-  "tests/hmac/sha256_test.vmem"
-  "tests/rv_timer/rv_timer_test.vmem"
+TEST_TARGETS=(
+  "tests/flash_ctrl/flash_test.vmem"
+  # XXX: Temporarially disabled, since these tests are failing at HEAD.
```

Comment:
nit: `Temporarily`.

Reply:
Please also add the issue number to the comment.
XXX -> TODO